### PR TITLE
typo + duplicate entries in tex-command list fixed

### PR DESCRIPTION
**Pull request by Debenben**
What I found going through the command list:
* \bul should be \bull
* \Space marked as non-standard
* some duplicate entries

### CODE REVIEW

File: tex.rst [hunk 1]
```diff
@@ -1190,13 +1190,11 @@ B
 
 .. code-block:: latex
 
-    \Bbb                    mediawiki-texvc
```
Comment (dpvc) [inline tex.rst]: `\Bbb` is in `mediawiki-texvc, but it is also in the core TeX input jax.  So I don't think this should be removed (as it does exist), but I could see removing the `mediawiki-texvc` entry.

Comment (pkra) [reply tex.rst]: It's just moved lower but duplicated there.

Maybe it should be removed from `mediawiki-texvc.js` in general?

**Comment by pkra** [comment]
> sorry, I am unsure how to handle the pull requests with the web interface. 

You can make a local change and push to your branch -- that will update the PR.

File: tex.rst (hunk 1 above)
Comment (dpvc) [reply tex.rst]: Argh!  I didn't see that.  I did a pretty poor job putting this list together originally, it seems.

Comment (dpvc) [reply tex.rst]: It probably should be removed from `mediawiki-texcv`.

**Pull request merged**